cmd/ocsp-updater/main.go

```diff
@@ -436,6 +489,7 @@ func main() {
 		go updater.newCertificatesLoop.loop()
 		go updater.oldOCSPResponsesLoop.loop()
 		go updater.missingSCTReceiptsLoop.loop()
+		go updater.revokedCertificatesLoop.loop()
```

Comment:
Better to put these looper objects in a slice and iterate over the slice, to make it harder to forget to start one.